tools/RazorPageGenerator/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "1.1.0-*",
```

Comment:
Yeah, for packages.csv make sure it's not a shipping package.

---
In reply to: [75167368](https://github.com/aspnet/Common/pull/139#discussion_r75167368) [](ancestors = 75167368)